src/utils.rs

```diff
@@ -8,57 +10,89 @@ use serenity::model::{
 use std::collections::HashMap;
 
 pub fn get_num_guilds(guilds: &Vec<GuildStatus>) -> String {
-    if guilds.len() == 1 {
-        String::from("1 guild")
-    } else {
-        format!("{:?} guilds", guilds.len())
-    }
+    format!(
+        "{} guild{}",
+        guilds.len(),
+        if guilds.len() > 1 { "" } else { "s" }
+    )
 }
 
 pub fn get_guild_human_members(members: &Vec<Member>) -> usize {
-    let human_members: Vec<Member> = members
-        .iter()
-        .filter(|member| !member.user.bot)
-        .map(|x| x.clone())
-        .collect();
-
-    human_members.len()
+    members.iter().filter(|member| !member.user.bot).count()
```

Comment:
`count` is more efficient than `collecting` the items, which requires allocating a new `Vec`